db/migrate/20160720151217_create_arbitration_rules.rb

```diff
@@ -0,0 +1,16 @@
+class CreateArbitrationRules < ActiveRecord::Migration[5.0]
+  def change
+    create_table :arbitration_rules do |t|
+      t.string :type
```

Comment:
type column is reserved for STI, are you planning to do STI.  If not you'll need a new column name.